packages/pnpm/package.json

```diff
@@ -93,6 +92,7 @@
     "p-defer": "^3.0.0",
     "path-exists": "^4.0.0",
     "path-name": "^1.0.0",
+    "pkg": "5.2.1",
```

Comment:
it won't work properly with plain pkg. I had to do some changes in the fork to make it work with pnpm.

Reply:
Sure. I wait for the fork to be updated.

Reply:
:ship: `@zkochan/pkg@0.0.0-2`

Reply:
Thanks"